phd-tests/tests/src/cpuid.rs

```diff
@@ -48,21 +50,11 @@ async fn cpuid_instance_spec_round_trip_test(ctx: &Framework) {
 }
 
 #[phd_testcase]
-async fn cpuid_boot_test(ctx: &Framework) {
+async fn cpuid_boot_and_migrate_test(ctx: &Framework) {
```

Comment:
should we, perhaps, leave the previous test in place, and add an additional test for the migration stuff? it seems like it's maybe worthwhile to have separate tests for "booting a VM with CPUID values works" and "CPUID values are not eaten on live migration" so we can detect those two failures separately?

on the other hand, setting up a PHD test is fairly expensive compared to e.g. typical unit tests, so maybe combining it into one test case is useful...I just felt like it was worth asking about.

Reply:
also, N.B. that the `phd_skip!` macro invocation a few lines down refers to the test by its previous name, maybe this should be updated? (can't comment on that line directly as it's not part of the diff)

Reply:
Good catch--I'll rearrange this a bit.

> (can't comment on that line directly as it's not part of the diff)

someday...

Reply:
5298e2a should help with this.